model/src/lib.rs

```diff
@@ -23,11 +24,21 @@ pub enum ClaimAvailabilityView {
 #[derive(Serialize, Deserialize, Debug, PartialEq)]
 #[serde(crate = "near_sdk::serde")]
 pub struct ClaimResultView {
-    pub total: U128,
+    pub asset: Asset,
+    pub is_success: bool,
+    pub total: Option<U128>,
 }
 
 impl ClaimResultView {
-    pub fn new(total: u128) -> Self {
-        Self { total: U128(total) }
+    pub fn new(asset: Asset, is_success: bool, total: Option<u128>) -> Self {
+        Self {
+            asset,
+            is_success,
+            total: total.map(U128),
+        }
     }
 }
+
+#[derive(Serialize, Deserialize, Debug, Default, PartialEq)]
+#[serde(crate = "near_sdk::serde")]
+pub struct ClaimAllResultView(pub Vec<ClaimResultView>);
```

Comment:
```suggestion
pub type ClaimAllResultView = Vec<ClaimResultView>;
```

After this change also adjust `on_asset_transfer` callback like this:
<img width="745" alt="image" src="https://github.com/sweatco/sweat-claim/assets/10122409/18a95a1a-53a7-4c74-9574-185148c19392">

And `ClaimAllResultViewExt` implementation:
<img width="577" alt="image" src="https://github.com/sweatco/sweat-claim/assets/10122409/e0a84ea5-a960-4fab-8c03-27cfd96c1f80">